Mage.Sets/src/mage/cards/v/VegaTheWatcher.java

```diff
@@ -0,0 +1,70 @@
+package mage.cards.v;
+
+import java.util.UUID;
+import mage.MageInt;
+import mage.abilities.common.SpellCastControllerTriggeredAbility;
+import mage.abilities.effects.Effect;
+import mage.abilities.effects.common.DrawCardSourceControllerEffect;
+import mage.constants.SubType;
+import mage.constants.SuperType;
+import mage.abilities.keyword.FlyingAbility;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.CardType;
+import mage.constants.Zone;
+import mage.game.Game;
+import mage.game.events.GameEvent;
+
+/**
+ *
+ * @author weirddan455
+ */
+public final class VegaTheWatcher extends CardImpl {
+
+    public VegaTheWatcher(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.CREATURE}, "{1}{W}{U}");
+
+        this.addSuperType(SuperType.LEGENDARY);
+        this.subtype.add(SubType.BIRD);
+        this.subtype.add(SubType.SPIRIT);
+        this.power = new MageInt(2);
+        this.toughness = new MageInt(2);
+
+        // Flying
+        this.addAbility(FlyingAbility.getInstance());
+
+        // Whenever you cast a spell from anywhere other than your hand, draw a card.
+        this.addAbility(new VegaTheWatcherTriggeredAbility(new DrawCardSourceControllerEffect(1), false));
+    }
+
+    private VegaTheWatcher(final VegaTheWatcher card) {
+        super(card);
+    }
+
+    @Override
+    public VegaTheWatcher copy() {
+        return new VegaTheWatcher(this);
+    }
+}
+
+class VegaTheWatcherTriggeredAbility extends SpellCastControllerTriggeredAbility {
+
+    public VegaTheWatcherTriggeredAbility(Effect effect, boolean optional) {
+        super(effect, optional);
+        this.rule = "Whenever you cast a spell from anywhere other than your hand, draw a card.";
+    }
+
+    public VegaTheWatcherTriggeredAbility(final VegaTheWatcherTriggeredAbility ability) {
```

Comment:
these copy constructors can be private since they're only called by the copy method

Reply:
I was wondering about that.  I see it about half the time as public and half private.  The class I extended had it as public so I used that but I'll keep in mind for next time.

Reply:
It's something I wanna refactor but I've never gotten around to it and I don't know if I ever will. Whenever I edit an old file I always make a point of correcting that but I still haven't automated it.